.circleci/config.yml

```diff
@@ -1157,7 +1157,7 @@ jobs:
       - run: pip3 install setuptools==49.6.0
       - run:
           command: |
-            sudo curl -Lfs https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip -o protoc3.zip
+            sudo curl -Lfs https://github.com/protocolbuffers/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip -o protoc3.zip
```

Comment:
nit: is it necessary to be downloading and unzipping as root here?

Reply:
It _should_ work without being sudo. 
This job is deprecated anyway and will be moved to Github Action where we wil not use sudo